documents/functionnal_specifications/functional.md

```diff
@@ -0,0 +1,445 @@
+|--------------|------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------|
+| **Car**      | Represents the environmental impact of personal and small vehicle transportation, including emissions and fuel consumption. Upgrades may focus on reducing carbon output, promoting electric vehicles, and increasing fuel efficiency. | ![car](pictures/transport/icons8-berline-60.png) |
+| **Bus/Truck** | Encompasses larger land vehicles used for public and freight transportation. This category emphasizes emissions reduction, fuel optimization, and transitioning to sustainable energy sources. | ![bus_truck](pictures/transport/icons8-navette-50.png) |
+| **Subway**   | Represents underground or metropolitan rail systems, which generally offer lower emissions per passenger. Improvements in this category could enhance energy efficiency and optimize route planning for reduced impact. | ![subway](pictures/transport/icons8-métro-48.png) |
+| **Boat**     | Focuses on the impact of water-based transport, from cargo ships to personal boats. Key areas include fuel emissions, wastewater management, and transitions to cleaner energy sources. | ![boat](pictures/transport/icons8-yacht-50.png) |
+| **Plane**    | Represents the environmental impact of air travel, which is a significant source of greenhouse gas emissions. Upgrades can include carbon offset programs, fuel efficiency improvements, and investments in alternative fuels. | ![plane](pictures/transport/icons8-décollage-d'avion-50.png) |
+
+<br>
+
+| Energy Systems           | Description                                                                                                          | Icon                                             |
+|--------------------------|----------------------------------------------------------------------------------------------------------------------|--------------------------------------------------|
+| **Coal Combustion**      | Electricity production using coal combustion, which results in high carbon emissions and pollution. Upgrades focus on reducing coal dependency and improving efficiency. | ![coal combustion](pictures/energy/icons8-charbon-de-bois-50.png) |
+| **Fossil Fuels**         | General energy production from fossil fuels, such as oil and coal, which contribute to greenhouse gas emissions. Improvements aim to reduce reliance on these non-renewable resources. | ![fossil fuels](pictures/energy/icons8-industrie-pétrolière-50.png) |
+| **Gas** | Electricity production using natural gas, which emits less CO₂ than coal but still has an environmental impact. Upgrades here focus on improving plant efficiency or transitioning to cleaner alternatives. | ![gas](pictures/energy/icons8-usine-50.png) |
```

Comment:
355 : replace `Upgrades here focus on improving` with `Improvements are primarily intended to improve`